core/editor/editor-loader.php

```diff
@@ -71,24 +91,30 @@ public function print_root_template() {
 	}
 
 	/**
-	 * Normalize script configs for enqueue and register methods.
+	 * Normalize script/style configs to enqueue and register methods.
 	 *
 	 * @param array $script_configs
+	 * @param string $type can be ['script', 'style']
 	 *
 	 * @return array
 	 */
-	private function normalize_script_configs( array $script_configs ) {
-		$default = [
+	private function normalize_asset_configs( array $script_configs, $type ) {
+		$additional_defaults = 'style' === $type ?
+			[ 'media' => 'all' ] :
+			[ 'in_footer' => true ];
+
+		$default = array_merge( [
 			'handle' => '',
 			'src' => '',
 			'deps' => [],
 			'version' => ELEMENTOR_VERSION,
 			'in_footer' => true,
-		];
+		], $additional_defaults );
 
 		$replacements = [
 			'{{ASSETS_URL}}' => ELEMENTOR_ASSETS_URL,
-			'{{SUFFIX}}' => ( Utils::is_script_debug() || Utils::is_elementor_tests() ) ? '' : '.min',
+			'{{MIN_SUFFIX}}' => ( Utils::is_script_debug() || Utils::is_elementor_tests() ) ? '' : '.min',
+			'{{DIR_SUFFIX}}' => is_rtl() ? '.rtl' : '',
```

Comment:
not sure about "dir". it usually means "directory"

Reply:
`dir: rtl`
min - minimum
dir - direction

if I make shortcut for min I should make shortcut for dir